### PR TITLE
Bundle Logcat needs the initial (beginning) request

### PR DESCRIPTION
**Pull request by jmatsu**
The current behavior (legacy) is a non-bundle logcat. Bundle logcat allows bundling multiple sets of Logcat lines with data consistency.  For example, Streamed Logcat and Retry with splitting into chunks are based on Bundle Logcat.

This PR introduces transaction-like instructions that enclose logcat content requests.

### CODE REVIEW

**Review by tnj — COMMENTED** [review]
Way to go!

File: sdk/src/main/java/com/deploygate/sdk/ClientId.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import android.text.TextUtils;
+
+import java.util.Locale;
+import java.util.UUID;
+
+class ClientId {
+    private static final String CLIENT_ID_PREFIX = "c:";
+
+    static String generate() {
+        return String.format(Locale.US, "%s:%s", CLIENT_ID_PREFIX, UUID.randomUUID().toString());
```
[the remark this answers is not in the document]
Comment (jmatsu) [reply sdk/src/main/java/com/deploygate/sdk/ClientId.java:12]: omg, I'll be remove `:` from the format string!

**Review by tnj — APPROVED** [review]
LGTM!

**Comment by jmatsu** [comment]
Thanks as always.

**Pull request merged**